L.Map.Sync.js

```diff
@@ -41,6 +48,27 @@
             return this;
         },
 
+        // Helper function to compute the offset easily
+        // The arguments are relative positions respect refence and target maps
```

Comment:
Sentence seems to miss some (parts of) words (added with **emphasis**):
  ... are relative positions **with** respect to ref**er**ense and target maps.